ui/perfherder/ProgressBar.jsx

```diff
@@ -7,14 +7,16 @@ import SimpleTooltip from '../shared/SimpleTooltip';
 const ProgressBar = ({ magnitude, regression, color }) => (
   <SimpleTooltip
     text={
-      <Progress multi>
+      <Progress multi aria-label="Magnitude of Difference">
         {/* the % of the bars that are colored and transparent is based on the newIsBetter metric,
         which determines whether the colored bar for magnitude is displayed on the left or right */}
+        <span className="sr-only sr-only-focusable">Magnitude 1</span>
```

Comment:
This is a good start but "Magnitude 1" doesn't tell the user anything useful. We'd want to display some sort of text that conveys the values with context/meaning ( like regression or improvement).

Reply:
hrm, perhaps we should rethink this. It might be more helpful to hide the progress bars from screen readers and on line 10 replace the aria-label text "Magnitude of Difference" with something like: "Magnitude of difference based on lower is better metric: % <magnitude> regressed" (or % improved - base it on the logic being used for the progress bar values).